client/app/admin/admin.component.ts

```diff
@@ -45,18 +56,74 @@ export class AdminComponent implements OnInit {
     constructor(
         private menuService: MenuService,
         private usersService: UsersService,
-        private tableService: TableService
-    ) {}
+        private formBuilder: FormBuilder,
+        protected orderSerice: OrderService,
+        private tableService: TableService,
+        private adminService: AdminService
+    ) { }
 
     /**
      * Get data on load
      */
     ngOnInit() {
         this.getMenu();
         this.getUsers();
+        this.getTotalForToday();
+        this.getTotalForLastWeek();
+        this.getTotalForLastMonth();
         this.getTables();
     }
 
+    /**
+    * Get total for last 24h
+    */
+    getTotalForToday() {
+        const timePeriod = { to: null, from: null };
+        timePeriod.to = new Date();
+        timePeriod.from = new Date(new Date().setDate(new Date().getDate() - 1));
```

Comment:
Does this need the outer Date?

Reply:
Do we want to get the total for the last 24 hours, or is it best to do it just for that day (i.e from midnight to now)?

Reply:
Google Analytics does it this way, and allows you to select custom date-times as well. I think we can just leave it like this for now, since is just an additional feature anyway